multicluster/controllers/multicluster/resourceexport_controller.go

```diff
@@ -89,6 +89,8 @@ func (r *ResourceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		klog.V(2).InfoS("Reconciling AntreaClusterNetworkPolicy type of ResourceExport", "resourceexport", req.NamespacedName)
 	case common.ClusterInfoKind:
 		return r.handleClusterInfo(ctx, req, resExport)
+	case common.LabelIdentityKind:
```

Comment:
I think you can add a filter during manger set up like lable_identity_export_controller here https://github.com/Dyanngg/antrea/commit/f0da0b07d180302de42786c401ba7cc150d506f9#diff-50aa24f85e552f16f968db4961a8e544e5f699dc510fb62702e0bb1dc6afc2f4R98 to exclude `LabelIdentityKind`.

Reply:
Done.